checkstyle/suppressions.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <!-- Clients -->
+    <suppress checks="ClassFanOutComplexity"
+              files="(Fetcher|ConsumerCoordinator|KafkaConsumer|KafkaProducer|SaslServerAuthenticator).java"/>
+    <suppress checks="ClassFanOutComplexity"
+              files=".*/protocol/Errors.java"/>
+
+    <suppress checks="MethodLength"
+              files="KerberosLogin.java"/>
+
+    <suppress checks="ParameterNumber"
+              files="NetworkClient.java"
+              lines="132"/>
+    <suppress checks="ParameterNumber"
+              files="KafkaConsumer.java"
+              lines="722"/>
+    <suppress checks="ParameterNumber"
+              files="ConsumerCoordinator.java"
+              lines="91"/>
```

Comment:
3 cases above where we have `lines` still. And a fourth one in `WorkerSourceTask`. Did you leave them intentionally?